packages/apollo-federation/src/service/printFederatedSchema.ts

```diff
@@ -332,30 +332,20 @@ function printDescription(
     | GraphQLEnumValue
     | GraphQLUnionType,
   indentation: string = '',
-  firstInBlock: boolean = true,
+  _firstInBlock: boolean = true,
```

Comment:
Seems like this isn't really needed? Lists with descriptions seem to work out fine, as you can see from the above tests.

Reply:
@JacksonKearl yeah let's remove this